app/src/main/java/com/example/composetemplate/presentation/common/LoginTextField.kt

```diff
@@ -49,24 +47,24 @@ fun LoginTextField(
                     else {
                         if (!isValid) Color.Red else Color.Transparent
                     }
-                ), CircleShape
+                ), CustomTheme.shapes.textField
             ),
         value = text,
         onValueChange = onValueChange,
         placeholder = {
-            Text(text = loginScreenEnum.getPlaceHolder(), color = Color.White)
+            Text(text = loginScreenEnum.getPlaceHolder(), color = CustomTheme.colors.text)
         },
-        shape = CircleShape,
+        shape = CustomTheme.shapes.textField,
         colors = TextFieldDefaults.colors(
-            cursorColor = if (isSystemInDarkTheme()) Color.White else Color.Black,
+            cursorColor = CustomTheme.colors.cursor,
             disabledIndicatorColor = Color.Transparent,
-            errorIndicatorColor = Color.Red,
+            errorIndicatorColor = CustomTheme.colors.error,
             focusedIndicatorColor = Color.Transparent,
             unfocusedIndicatorColor = Color.Transparent,
-            unfocusedContainerColor = Color.DarkGray,
-            focusedContainerColor = Color.DarkGray,
-            focusedTextColor = LoginColorEnable,
-            unfocusedTextColor = if (isValid) LoginColorEnable else Color.Red
+            unfocusedContainerColor = CustomTheme.colors.editTextContainer,
```

Comment:
same is here, this is edittext of login, maybe naming can change. editTextContainer can be for other edit text in app